Makefile

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+SHELL = bash
+
+PYTHON_SRCS=$(shell find . -name "*.py" -not -path "./env/*" -not -path "./symbiflow/*")
+JSON_SRCS=$(shell find . -name "*.json" -not -path "./env/*" -not -path "./symbiflow/*")
+VERILOG_SRCS=$(shell find . -name "*.v" -not -path "./env/*" -not -path "./symbiflow/*")
+
+env:
+	pip install -r requirements.txt
```

Comment:
Probably should use make-env here -- but that can wait until a later pull request.